server.js

```diff
@@ -15,8 +15,17 @@ var checkVersion = function(major, minor, patch) {
     return true;
 }
 
+
+// Only print if debug mode is on
+var log = function(string, error) {
```

Comment:
The `error` parameter is a bit unclear when calling the function. I would prefer something like this:

```
var log = {
  debug: function (message) {
    if (debug) log.error(message);
  },
  error: function (message) {
    console.info(message);
    fs.write('phantomjs.log', message + '\n', 'a');
  }
};
```

And then use either `log.debug` or `log.error`. (Not sure whether you can refer to the `log` variable itself while it is being define.)